### PR TITLE
Use ParserKeyword& instead of ParserKeyword*

### PR DESCRIPTION
**Pull request by joakim-hove**
Downstream of: https://github.com/OPM/opm-common/pull/1005

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-common=1005 please

File: opm/upscaling/CornerpointChopper.hpp [hunk 1]
```diff
@@ -379,7 +379,7 @@ namespace Opm
             if (data.empty())
                 return;
 
-            Opm::DeckKeyword dataKw(&keyword);
+            Opm::DeckKeyword dataKw(keyword);
```
Comment (bska) [inline opm/upscaling/CornerpointChopper.hpp]: You probably need to make the same change on [line 359](https://github.com/OPM/opm-upscaling/blob/1bc89ef7aae6265bd149be8a244c183c44a853ab/opm/upscaling/CornerpointChopper.hpp#L359) (in the `addDoubleKeyword()` member function).

Comment (joakim-hove) [reply opm/upscaling/CornerpointChopper.hpp]: Yes - thank you - done. My local opm-upscaling did not build at all - some lapack issue????

**Comment by joakim-hove** [comment]
jenkins build this opm-common=1005 please

**Comment by bska** [comment]
Hm, the build request didn't take.  I'll try to relaunch it.

**Comment by bska** [comment]
jenkins build this opm-common=1005 please

**Comment by joakim-hove** [comment]
@akva2 : Can this be merged?

**Comment by akva2** [comment]
merging this broke the build https://ci.opm-project.org/job/opm-upscaling/9/ as the upstream has not been merged.

**Comment by joakim-hove** [comment]
> merging this broke the build https://ci.opm-project.org/job/opm-upscaling/9/ as the upstream has not been merged.

Hmmm - are you sure? The rebased upstream: https://github.com/OPM/opm-common/pull/1042 was merged.

**Comment by akva2** [comment]
okay, maybe unfortunate timing then. i'll rerun to check.

**Comment by akva2** [comment]
do remember to close the original PR if there is nothing left in there.

**Comment by akva2** [comment]
okay, build is okay. follow the way of the river and we avoid this in the future :)

**Pull request merged**